examples/linear_model/plot_ols.py

```diff
@@ -52,7 +52,7 @@
 print('Coefficients: \n', regr.coef_)
 # The mean square error
 print("Residual sum of squares: %.2f"
```

Comment:
line above says "The mean square error". I would just replace "Residual sum of squares" by "Mean squared error"
and add the "d" at the end of square above.